nestkernel/recording_backend_sionlib.h

```diff
@@ -75,12 +80,41 @@ filename is composed.
 Data format
 +++++++++++
 
-The container files contain the data in a custom binary format. The
-recorder property ``label`` and all other recorder-specific metadata
-is stored with each recording device. To retrieve the data after the
-simulation, the `nestio reader API` in Python has to be used. SIONlib
-container files are sparse files: They only use up the space on disk
-which is actually filled with data.
+In contrast to other recording backends, the ``sionlib`` backend
+writes the data from all recorders using it to a single container
+file(s). The file(s) contain the data in a custom binary format, which
+is composed of a series of blocks in the following order:
+
+* the *body block* contains the actual data records; the layout of an
+  individual record depends on the type of the device and is described
+  by a corresponding entry in the *device info block*
+  
+* the *file info block* keeps the file's metadata like version
+  information and such
+
+* the *device info block* stores the properties and a data layout
+  description for each device that uses the ``sionlib`` backend
+
+* the *tail block* contains pointers to the *file info block*.
```

Comment:
@jougs bcaae82f9a1a6d45477f0a232db6f9a65e394dc3 both removes periods, and adds a new one. so it's still inconsistent. Also, I still think the bullet points here should start with a capital letter (at least according to [this guide](https://www.lexico.com/grammar/bullet-points)).

```suggestion
* The *body block* contains the actual data records; the layout of an
  individual record depends on the type of the device and is described
  by a corresponding entry in the *device info block*
  
* The *file info block* keeps the file's metadata, like version
  information and such

* The *device info block* stores the properties and a data layout
  description for each device that uses the ``sionlib`` backend

* The *tail block* contains pointers to the *file info block*
```

Reply:
Agreed!